src/libs/address.js

```diff
@@ -18,13 +23,19 @@ export function format(address) {
       address.country,
     ]
       .filter(i => i)
-      .filter((item, pos, arr) => pos === 0 || item !== arr[pos - 1]) // remove consecutive duplicated name
-      .join(', ');
+      .filter((item, pos, arr) => pos === 0 || item !== arr[pos - 1]); // remove consecutive duplicated name
   }
 
-  return [address.street, address.city, address.country]
-    .filter(i => i) // Filter out any undefined value
-    .join(', ');
+  return [address.street, cityAndPostcode, address.country]
+    .filter(i => i); // Filter out any undefined value
+}
+/**
+ * Format an address given an address object (name, city, country, label, and other regions)
+ * @param {*} address - an address object
+ * @param string separator - how items are joined
+ */
+export function format(address, separator = ', ') {
```

Comment:
I think this function isn't used anymore in the code base.

Reply:
actually it is, but renamed (I'm not the author, is that really necessary?)
![image](https://user-images.githubusercontent.com/1225909/88637374-60845080-d0ba-11ea-80c2-1e84b3d84f61.png)

Reply:
Good catch! Sorry I searched for `format(` calls and forgot the function could be renamed on import :)
But this raises another question: you modified this function behavior in `toArray`, does it still match what we expect in `SuggestItem`?

Reply:
The edits I made in address.js only introduce a new intermediary toArray function and adds the postcode in some cases.
According to my tests, the suggest items haven't been altered by these edits:
If I search an address, or a city, or a PoI, the addresses inside the suggestion items won't contain postcodes, so nothing changed compared to before imo.